lib/attack.go

```diff
@@ -71,8 +71,8 @@ func NewAttacker(opts ...func(*Attacker)) *Attacker {
 
 	a.client = http.Client{
 		Transport: &http.Transport{
-			Proxy:                 http.ProxyFromEnvironment,
-			Dial:                  a.dialer.Dial,
+			Proxy: http.ProxyFromEnvironment,
+			Dial:  a.dialer.Dial,
```

Comment:
Looks like `go fmt` should be used here ?

Reply:
My editor runs `goimports` on save. I imagine this file hasn't been fmt'd since the go1.11 formatting changes.

Reply:
Yeah, you are right ;)

Reply:
This turned out to be my bad, my goimports binary was _ancient_. I only realised when it choked on the type alias :-(